### PR TITLE
test: add e2e deployer test

### PR DESCRIPTION
**Pull request by SweeXordious**
## Overview

Closes https://github.com/celestiaorg/orchestrator-relayer/issues/167

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: e2e/deployer_test.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	_, err = network.GetLatestDeployedQGBContractWithCustomTimeout(ctx, 15*time.Second)
+	HandleNetworkError(t, network, err, true)
+
+	err = network.DeployQGBContract()
+	HandleNetworkError(t, network, err, false)
+
+	bridge, err := network.GetLatestDeployedQGBContract(ctx)
+	HandleNetworkError(t, network, err, false)
+
+	// FIXME should we use the evm client here or go for raw queries?
```
Comment (SweeXordious) [inline e2e/deployer_test.go:42]:
```suggestion
```

**Comment by evan-forbes** [comment]
is this blocked by #192? or should we be expecting the CI to fail here?

**Comment by SweeXordious** [comment]
Yes, it's  blocked by 192. Will merge now and update the branch

File: e2e/deployer_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/orchestrator-relayer/evm"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDeployer(t *testing.T) {
```
Comment (evan-forbes) [inline e2e/deployer_test.go:15]: ref #205

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**